### PR TITLE
Update permissions.yml cilvipeterparker

### PR DESCRIPTION
**Pull request by SirRicardo**

### CODE REVIEW

**Comment by ItzHarkomal** [comment]
There should be only two spaces before cilvipeterparker

**Comment by ItzHarkomal** [comment]
Change §'s in prefix to &

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -8566,6 +8566,34 @@ groups:
...
+    options:
+      prefix: '&4Civilian&fPe&4ter&0P&4ar&0k&4er'
+      suffix: '&4'
+    worlds:
+      survival:
+        permissions:
+        - powers.wallclimb
+        - ch.alias.websling
+        - -essentials.protect.damage.fall
+        - ch.alias.spidermanpeterparker
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this is not spiderman peter parker

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -8566,6 +8566,34 @@ groups:
         - ch.alias.haste1
         - ch.alias.haste2
         - ch.alias.haste3
+  civilianpeterparker:
+    inheritance:
+    - commonherotraits
+    options:
+      prefix: '&4Civilian&fPe&4ter&0P&4ar&0k&4er'
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: The name should just be Peter Parker

**Pull request closed without merging**